pkg/app/ops/insightcollector/application_collector.go

```diff
@@ -88,13 +97,7 @@ func (c *applicationDataCollector) listApplications(ctx context.Context) ([]*mod
 
 	for {
 		apps, next, err := c.lister.List(ctx, datastore.ListOptions{
-			Filters: []datastore.ListFilter{
-				{
-					Field:    "Deleted",
-					Operator: datastore.OperatorEqual,
-					Value:    false,
-				},
-			},
```

Comment:
Sorry, I don't understand why do we have to remove this filter value from the query, could you explain?